src/motile_tracker/data_views/views/ortho_views.py

```diff
@@ -0,0 +1,263 @@
+            layer and process them there.
+    """
+
+    def sync_paint(orig_layer: TrackLabels, copied_layer: Labels, event: Event):
+        """Process paint event on original TrackLabels instance."""
+
+        if copied_layer.data.ndim > 3:
+            orig_layer._on_paint(event)
+        else:
+            show_info("Painting in the time dimension is not supported")
+            orig_layer._revert_paint(event, copied_layer)
+            orig_layer.refresh()
+
+    def paint_wrapper(event: Event):
+        """Wrap paint event and send to original layer."""
+
+        return sync_paint(orig_layer, copied_layer, event)
+
+    copied_layer.events.paint.connect(paint_wrapper)
+
+
+def track_layers_hook(
+    orig_layer: TrackLabels | TrackPoints, copied_layer: Labels | Points
+) -> None:
+    """Hook to capture click events on TrackLabels and TrackPoints derived Labels and
+    Points layers, and forward them to their original layer. Also, register key binds
+    for view mode, undo & redo to copied layer, that call functions on the original layer.
+
+    Args:
+        orig_layer (TrackLabels | TrackPoints): TracksLabels or TrackPoints layer from
+            which the copied layer is derived.
+        copied_layer (Labels | Points): Labels or Points equivalent of the TracksLabels
+            or TrackPoints layer.
+    """
+
+    # define the click behavior the layer should respond to
+    def click(
+        orig_layer: TrackLabels | TrackPoints, layer: Labels | Points, event: Event
+    ):
+        if layer.mode == "pan_zoom":
+            mouse_press_time = time.time()
+            dragged = False
+            yield  # start
+            while event.type == "mouse_move":
+                dragged = True
+                yield
+            if dragged and time.time() - mouse_press_time < 0.5:
+                dragged = False  # treat micro drags as clicks
+
+            if not dragged:
+                if isinstance(layer, Labels):
+                    label = layer.get_value(
+                        event.position,
+                        view_direction=event.view_direction,
+                        dims_displayed=event.dims_displayed,
+                        world=True,
+                    )
+                    orig_layer.process_click(event, label)
+
+                if isinstance(layer, Points):
+                    point_index = layer.get_value(
+                        event.position,
+                        view_direction=event.view_direction,
+                        dims_displayed=event.dims_displayed,
+                        world=True,
+                    )
+                    orig_layer.process_point_click(point_index, event)
+
+    # Wrap and attach click callback
+    def click_wrapper(layer, event):
+        return click(orig_layer, layer, event)
+
+    copied_layer.mouse_drag_callbacks.append(click_wrapper)
+
+    # Bind keys to original layer TracksViewer
```

Comment:
Did we define these keys somewhere? I remember objecting to the duplication of these keybinds before... but I also remember saying "oh well it's not important right now", so I can't recall what if anything happened. If it isn't defined anywhere, perhaps it's time to make a proper issue and add it to the roadmap.